tests/ieee802154_submac/Makefile

```diff
@@ -20,8 +20,12 @@ USEMODULE += od
 USEMODULE += shell
```

Comment:
Can you include `shell_commands`, otherwise I don't see the interes of including `ps`